content/v2.0/actions/request-and-response.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Request & Response
+order: 40
+---
+
+## Request
+
+In order to access the metadata coming from a HTTP request, an action has a private object `request` that derives from `Rack::Request`.
```

Comment:
```suggestion
In order to access the metadata coming from an HTTP request, an action has a private object called `request`.
```

Feels like too much info about implementation details so better to keep it simple.